hw/ip/aes/dv/aes_sim.core

```diff
@@ -20,9 +20,21 @@ filesets:
     file_type: systemVerilogSource
 
 targets:
-  sim:
+  sim: &sim_target
```

Comment:
Thanks for updating it Michael. Do you mind also help us update the dvgen template here: https://github.com/lowRISC/opentitan/blob/master/util/uvmdvgen/sim.core.tpl#L23

Reply:
Good point, thanks @cindychip. Updated!